### DISCUSSION TITLE
Override Expander Style

### DISCUSSION THREAD

**Discussion by Progiscad** [MahApps/MahApps.Metro#4509] [General]
Hello
I would like to change the style of the expander's header. I would like to remove its rectangular appearance with a border and its CornerRadius property
![image](https://github.com/user-attachments/assets/4b2aafcb-361f-47e2-93c0-5b604699f497)

Can you give me a style idea to do this?
Thank you

**Comment by timunie** [comment]
You should be able to set `mah:ControlsHelper.CornerRadius` (at least in upcomming v3): 
https://github.com/MahApps/MahApps.Metro/blob/f7ba30586e9670f07c2f7b6553d129a9e32fc673/src/MahApps.Metro/Styles/Controls.Expander.xaml#L350
